src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
```

Comment:
Similarly, using `pop` on the `toFormat` array mutates the original array. Consider accessing the last element by index instead of using `pop`.